source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -1168,6 +1177,7 @@ void RtFileIo::write(std::ostream& stream, const RtWriteOptions* options)
     if (options)
     {
         xmlWriteOptions.writeXIncludeEnable = options->writeIncludes;
+        document->setVersionString(options->desiredMajorVersion + "." + options->desiredMinorVersion);
```

Comment:
Trivial thing: there is a makeVersionString() method in Util. Does exactly the same thing to concat the string together.